app/services/foreman_rh_cloud/cloud_connector.rb

```diff
@@ -0,0 +1,68 @@
+require 'securerandom'
+
+module ForemanRhCloud
+  class CloudConnector
+
+    CLOUD_CONNECTOR_USER = 'cloud_connector_user'.freeze
+    CLOUD_CONNECTOR_TOKEN_NAME = 'Cloud connector access token'.freeze
+    CLOUD_CONNECTOR_FEATURE = 'ansible_configure_cloud_connector'.freeze
+
+    def install
+      user = service_user
+      token_value = personal_access_token(user)
+
+      composer = ::JobInvocationComposer.for_feature(
+        CLOUD_CONNECTORE_FEATURE,
```

Comment:
Should be CLOUD_CONNECTOR_FEATURE instead of CLOUD_CONNECTOR**E**_FEATURE ( without an E in CONNECTOR)